src/icons/fileIcons.ts

```diff
@@ -895,6 +895,8 @@ export const fileIcons: FileIcons = {
         { name: 'dhall', fileExtensions: ['dhall', 'dhallb'] },
         { name: 'sml', fileExtensions: ['sml', 'mlton', 'mlb', 'sig', 'fun', 'cm', 'lex', 'use', 'grm'] },
         { name: 'nrwl', fileNames: ['nx.json'] },
+        { name: 'tsconfig', fileNames: ['tsconfig.json'] },
+        { name: 'jsconfig', fileNames: ['jsconfig.json'] },
```

Comment:
This also needs `fileExtensions`, since `foo.tsconfig.json` is a semi‑common way to differentiate multiple **TypeScript** configurations.

The other is `tsconfig.foo.json`.

```suggestion
        { name: 'tsconfig', fileNames: ['tsconfig.json'], fileExtensions: ['tsconfig.json'] },
        { name: 'jsconfig', fileNames: ['jsconfig.json'], fileExtensions: ['jsconfig.json'] },
```